constants.py

```diff
@@ -2,8 +2,11 @@
 
 QTOPCONF_YAML = 'qtopconf.yaml'
 QTOP_LOGFILE = '$HOME/.local/qtop/logs/qtop.log'
+QTOP_TARFN = 'qtop_tar_$USER.tar'
 # QTOP_LOGFILE = '$HOME/.local/qtop/logs/qtop_%s.log'  % os.getpid()
 QTOP_LOGFILE = os.path.expandvars(QTOP_LOGFILE)
+QTOP_TARFN = os.path.expandvars(QTOP_TARFN)
+savepath = os.path.expandvars('/tmp/qtop_results_$USER')
```

Comment:
also, savepath might also need $DATE_$TIME (think of the same user running it twice in parallel)